app/tag-management/all-tags/tag-list/tag-list.component.html

```diff
@@ -0,0 +1,22 @@
+<ol class="tag-list">
+  <li *ngFor="let tag of tags; let isLast = last">
+    <div class="tag-header">
+      <h3>{{ tag.name }}</h3>
+      <div *ngIf="tagsEditable" class="tag-actions">
+        <a [routerLink]="['/tags/edit', tag.id]"><md-icon title="edit tag">edit</md-icon></a>
+        <a [routerLink]="['/tags/delete', tag.id]"><md-icon title="delete tag">delete</md-icon></a>
+        <a [routerLink]="['/tags/add-sub-tag', tag.id]"><i class="material-icons">add</i></a>
+        <a [routerLink]="['/tags/remove-sub-tag', tag.id]"><i class="material-icons">clear</i></a>
```

Comment:
It's  not really clear, what this icon means in this context.
Maybe just showing this on subtags for delete that tag from the tag as subtag. Then we don't need any "do you really want" page for deleting subTag, since we can easily restore it by adding it again.